dimagi/pages/templates/sections/commcare/product_onboarding/pricing/heading.html

```diff
@@ -0,0 +1,30 @@
+<div class="feature-row border-top accent-gray-bg">
+  <div class="grid-4 grid-valign-middle gap-bottom-line pad-top-line">
+    <div class="column">
+    </div>{# .column #}
+    <div class="column">
+      <h5 class="align-center uppercase green font-weight-medium">
```

Comment:
Echoing the previous comment I made about the double `<h5>` tags and a better semantic change.